ui/pages/confirmations/components/confirm/info/shared/gas-fee-token-icon/gas-fee-token-icon.test.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+
+import { NATIVE_TOKEN_ADDRESS } from '../../../../../../../../shared/constants/transaction';
```

Comment:
Most of the files in the PR are only change in import path for `NATIVE_TOKEN_ADDRESS`.